docs/src/solvers/dde_solve.md

```diff
@@ -29,15 +28,15 @@ handle most problems. For fast solving at where non-strict error control is
 needed, choosing `OrwenZen3()` can do well. Using `BS3` is similar to the MATLAB
 `dde23`, but `OrwenZen3()` will have noticably less error for the same work.
 For algorithms where strict error control is needed, it is recommended that one
-uses `DP8()`. Other high order integrators are not applicable since they use
-a lazy interpolant.
+uses `DP8()`.
 
-For state-dependent delays, the current best choice is `RK4` since it uses a
-residual control method to more accurately step. Note that current state-dependent
-delays are not detected and thus non-residual control methods will be less
-accurate. Still, residual control is an error-prone method. We recommend setting
-the tolerances low (`1e-10`) and only trusting the solution to a 2-3 decimal
-places of accuracy.
+If state-dependent delays are declared, the algorithm will detect
+discontinuities arising from these delays and adjust the step size such that
+these discontinuities are included in the mesh, using the principle outlined by
+S. P. Corwin, D. Sarafyan and S. Thompson in "DKLAG6: a code based on
```

Comment:
No need to mention the citation here. This should instead get added to the citation page: https://github.com/JuliaDiffEq/juliadiffeq.github.io/blob/master/citing.md